olp-cpp-sdk-core/include/olp/core/network2/NetworkTypes.h

```diff
@@ -65,13 +67,15 @@ enum class ErrorCode {
  * trigger failed.  The caller must check whether the outcome of the request was
  * a success before attempting to access the result or the error.
  */
-class NetworkStatus {
+class SendOutcome final {
  public:
-  NetworkStatus(RequestId id) : request_id_(id) {}
-  NetworkStatus(ErrorCode code) : error_code_(code) {}
+  SendOutcome(RequestId request_id) : request_id_(request_id) {}
```

Comment:
```suggestion
  explicit SendOutcome(RequestId request_id) : request_id_(request_id) {}
```

Make it explicit in order to avoid implicit casts from uint.